ert/storage/_storage.py

```diff
@@ -563,10 +567,10 @@ def add_ensemble_record(
             f"non-existing experiment: {experiment_name}"
         )
 
-    dataframe = pd.DataFrame([r.data for r in ensemble_record.records])
+    dataframe = pd.DataFrame([r.data for r in ensemble_records])
```

Comment:
I think if you keep this the way it was 

`dataframe = pd.DataFrame([r.data for r in ensemble_records.records])`

There is no more need to check `is_invarient` on ln 381 in `_storage.py`. But it is up to you if you think the suggestion is ok or not.

Reply:
I think you are correct, I think not think of that option. `is_invariant` can then be eliminated.